src/checks/README.md

```diff
@@ -12,6 +12,10 @@ A Pokémon is considered to be legal iff all checks pass.
 
 A description of what the check is testing for. This will get reported as the failure reason if the check fails.
 
+* `section: string`
```

Comment:
This shouldn't block the PR (we've done enough bikeshedding already for this topic) but I feel like "sections" is kind of a vague term. Maybe we should use something like `illegalField` instead?